keel-deliveryconfig-plugin/keel-deliveryconfig-plugin.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
   implementation project(":keel-retrofit")
   implementation project(":keel-plugin")
   implementation project(":keel-front50")
+  implementation "javax.inject:javax.inject:1"
```

Comment:
needed for `Provider<BeanType>` to break circular dependency

spring probably has a native equivalent but I somewhat prefer the generic annotations

Reply:
sure, dropped this and switched to spring ObjectFactory which is the equivalent